tests/v1/ApiDumper/run_test

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env php
+<?php
+declare(strict_types=1);
+error_reporting(E_ALL);
+
+use Neighborhoods\Buphalo\V1\Api\FabricationFile;
+use Neighborhoods\Buphalo\V1\Api\FabricationFile\Actor\AnnotationProcessor;
+use Neighborhoods\Buphalo\V1\Api\Dumper;
+use Neighborhoods\Buphalo\V1\StringMap;
+use Neighborhoods\Buphalo\V1\AnnotationProcessors;
+
+$PWD = __DIR__;
```

Comment:
nittiest of nits:

pwd is the verb-phrase. i'd argue this should be either `$WD` or `$CWD`
but maybe `$PWD` is a standard and when in rome?